modules/apps/portal/portal-messaging/src/main/java/com/liferay/portal/messaging/internal/BaseAsyncDestination.java

```diff
@@ -119,12 +119,7 @@ public void open() {
 				new NamedThreadFactory(
 					getName(), Thread.NORM_PRIORITY,
 					PortalClassLoaderUtil.getClassLoader()),
-				(r, executor) -> {
-					_rejectedTaskCounter.incrementAndGet();
-
-					_rejectedExecutionHandler.rejectedExecution(r, executor);
-				},
-				new ThreadPoolHandlerAdapter());
```

Comment:
doesn't this lose counting when a rejection handler is provided?

Reply:
You are right! Sorry, forget user can set a different one, I will add a commit to keep the wrapping logic when user does set one.